### PR TITLE
Added test cases for RodCuttingTest.java

### PR DESCRIPTION
**Pull request by sunilnitdgp**
- [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [ ] This pull request is all my own work -- I have not plagiarized it.
- [ ] All filenames are in PascalCase.
- [ ] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [ ] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

### CODE REVIEW

**Comment by sunilnitdgp** [comment]
I request the maintainer(s) to merge my request using "hacktoberfest-accepted" label.

**Review by vil02 — CHANGES_REQUESTED** [review]
Please:
- remove the `main` method,
- mark `RodCutting` as `final`,
- add a `private` default constructor of `RodCutting`.

**Comment by siriak** [comment]
Linter is still failing

**Comment by sunilnitdgp** [comment]
@siriak Fixed the linter issue.

**Comment by sunilnitdgp** [comment]
@vil02 / @siriak Please review this and merge if everything looks good.

File: src/test/java/com/thealgorithms/dynamicprogramming/RodCuttingTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        int[] emptyPrice = {};
+        assertEquals(0, RodCutting.cutRod(emptyPrice, 0));
+    }
+
+    @Test
+    public void testCutRodWithLargeInput() {
+        // Test case 10: Large rod length
+        int[] price7 = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        assertEquals(10, RodCutting.cutRod(price7, 10));
+    }
```
[the remark this answers is not in the document]
Comment (vil02) [reply src/test/java/com/thealgorithms/dynamicprogramming/RodCuttingTest.java:71]: @sunilnitdgp this is still pending.

[the remark this answers is not in the document]
Comment (siriak) [reply src/test/java/com/thealgorithms/dynamicprogramming/RodCuttingTest.java:71]: Please provide a source of the test data or the method used to create it

**Pull request closed without merging**